sections/debt/components/ManageTab/index.tsx

```diff
@@ -1,12 +1,34 @@
+import Wei from '@synthetixio/wei';
+import useUserStakingData from 'hooks/useUserStakingData';
+import { useRecoilValue } from 'recoil';
+import { walletAddressState } from 'store/wallet';
 import styled from 'styled-components';
-
 import { FlexDivColCentered } from 'styles/common';
 
 const ManageTab = () => {
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { debtBalance: actualDebt } = useUserStakingData(walletAddress);
+
+	const uniswapURLWithQueryParams = (debt: Wei) =>
+		`https://app.uniswap.org/#/swap?inputCurrency=0x57ab1ec28d129707052df4df418d58a2d46d5f51&exactAmount=${debt.toNumber()}&exactField=input`;
 	return (
 		<ManageContainer>
-			Debt mirror
-			<ManageSubtitle>Coming soon</ManageSubtitle>
+			{!walletAddress ? (
+				<span>Please connect a wallet</span>
+			) : (
+				<iframe
```

Comment:
Not sure why it's important to add a title attribute to the iframe, but lets do it to avoid this warning